### DISCUSSION TITLE
Пожелания по документации

### DISCUSSION THREAD

**Discussion by Ge1i0N** [moonshine-software/moonshine#1207] [Ideas]
Две фичи могут сделать документацию более дружелюбной.

1. **Сохранение локализации**
Сейчас в рамках одной сессии при переключении на русский язык перевод сохраняется, однако периодически приходится всё же руками возвращать его, когда сессия истекает. Не критично, но не очень приятно. Возможно стоит вынести выбранную локаль в куки?

2. **Адаптив на "средних" экранах**
На больших размерах экрана к левому меню вопросов нет. На маленьких - тоже - оно уезжает туда, куда ожидается на мобилках.
А вот на средних экранах он вроде есть, но пользоваться им достаточно неудобно, может что-то можно придумать - например разворачивать по наведению - чтобы можно было посмотреть на все пункты и выбрать нужный, а не только всплывающим тайтлом показывать на что наведён курсор.
![Снимок экрана от 2024-08-26 14-34-39](https://github.com/user-attachments/assets/485b2bab-0625-4e09-84d0-809779fe930b)

**Comment by lee-to** [comment]
Спасибо! Прислушаемся! Можно продолжать этот топик

**Comment by ramir1** [comment]
3) Судя по чату достаточно много одинатовых вопросов вида "Как сделать ...", можно оформить отдельный FAQ с короткими ответами и ссылкой на документацию.

**Comment by ArtemYurov** [comment]
> 1. Сохранение локализации. Не критично, но не очень приятно.
Очень неприятно. Приходится всегда переходить по ссылке с установленным языком

> 4. Нужен работающий поиск по тексту в коде чтобы находились все примеры использования какой-либо функции

**Comment by ramir1** [comment]
хотелось бы в документации видеть свойства и методы - что они делают и какие есть. В чате много одинаковых вопросов по этому поводу.

**Reply by ArtemYurov** [reply]
Да, поэтому возник вопрос, а есть ли инструменты которые создают документацию со список всех методов на основе кода?
Типа такого 
https://blog.quickadminpanel.com/laravel-api-documentation-with-openapiswagger/
